src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -3,9 +3,8 @@
 import json
 import logging
 import os
-
```

Comment:
Please don't remove these empty lines. They separate the import sections (builtin, 3rd party and project-internal).